src/Templater.js

```diff
@@ -30,16 +30,16 @@ export default class Templater {
         return {
             trans(lang, key, variables) {
 
-                if (typeof variables === 'undefined') {
+                if (_.isUndefined(variables)) {
 
                     // support for trans(key, variable)
-                    if (typeof key === 'object') {
+                    if (_.isObjectLike(key)) {
```

Comment:
Do we support an array of keys?
If not, prob use `_.isPlainObject`.

Reply:
also add negative test case for array of keys if ever needed

Reply:
Hmmm... Right. I don't think we support arrays, based on the README. Will change asap.

Reply:
I don't think we have to cover the case when `key` is an array since that is a direct misuse of the library.